cobigen-cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -421,10 +420,14 @@ private void setOutputRootPath(File inputProject) {
                     logger.info("Exact match found: " + artifactDescription + ".");
                     userSelection.add(possibleArtifacts.get(0));
                     return userSelection;
+                }else if(possibleArtifacts.size()<1) {
+                	logger. info("No increment with that name has been found, Please provide correct increment name and try again ! Thank you");
+                       
+                        System.exit(1);
                 }
-
-                logger.info("Please enter the number(s) of " + artifactType
-                    + "(s) that you want to generate separated by comma.");
+                
+                /*logger.info("Please enter the number(s) of " + artifactType
+                    + "(s) that you want to generate separated by comma.");*/
```

Comment:
Can you explain me why have you removed this message?

Reply:
Actually this message is comes under If String representation is given,so dont need this message here .

Reply:
You are right that it comes under String representation, but please don't remove this message because in this part of the code (line 415):

```Java
if (possibleArtifacts.size() > 1) {
    printFoundArtifacts(possibleArtifacts, isIncrements, artifactType);
} 
```
We are printing all the found artifacts by number, like this:

```
(0) All
(1) TO's
(2) CRUD Rest services
...
```
Therefore that message makes sense, because the user needs to use numbers to select the option.